### PR TITLE
Add a failing Aruba-based RSpec test for the CLI client output

### PR DESCRIPTION
**Pull request by olleolleolle**
This Draft PR is about showing what [Aruba](https://github.com/cucumber/aruba/) can do for testing the CLI client output.

## TODO

- [x] Get some known-good output to assert against

### CODE REVIEW

**Comment by olleolleolle** [comment]
@bronzdoc Now, I installed a Gemstash 1.1.0 and checked its `--version` output.

This PR now adds a CLI output assertion.

File: spec/gemstash/cli_spec.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+RSpec.describe 'CLI executable', type: :aruba do
+  before do
+    run_command('exe/gemstash --version')
+  end
+
+  # Substring
```
Comment (bronzdoc) [inline spec/gemstash/cli_spec.rb]: Is this comment necessary?

Comment (olleolleolle) [reply spec/gemstash/cli_spec.rb]: Thanks for the sharp eye, I removed that and force-pushed.

**Comment by olleolleolle** [comment]
@bundlerbot r=[@bronzdoc]

**Pull request merged**